drivers/video/video_esp32_dvp.c

```diff
@@ -389,6 +389,34 @@ static int video_esp32_init(const struct device *dev)
 	return 0;
 }
 
+int video_esp32_set_selection(const struct device *dev, struct video_selection *sel)
+{
+	struct video_esp32_data *data = dev->data;
+	const struct video_esp32_config *cfg = dev->config;
+	int ret;
+
+	ret = video_set_selection(cfg->source_dev, sel);
+	if (ret) {
```

Comment:
If we wish to have MISRA compliance on Zephyr Video area, we can use `ret != 0` or `ret < 0` for Zephyr rule 85 [here](https://docs.zephyrproject.org/latest/contribute/coding_guidelines/index.html).